azure-pipelines.yml

```diff
@@ -79,16 +79,19 @@ strategy:
     'OpenJDK11 verify':
       image: 'ubuntu-20.04'
       cmd: "mvn -e --no-transfer-progress verify"
+    - cmd: "./.ci/travis.sh git-diff"
 
     # MacOS JDK11 verify
     'MacOS JDK11 verify':
       image: 'macOS-10.15'
       cmd: "JAVA_HOME=$JAVA_HOME_11_X64 mvn -e --no-transfer-progress verify"
+    - cmd: "./.ci/travis.sh git-diff"
```

Comment:
This is override of cmd variable, but you should do to this in addition

Reply:
I saw another YAML file i.e. appveyor.yml and it can easily override but It is showing "sequence is not accepted".
I tried to google and tried all the possible ways to lint YAML and it passes but when it is executing on azure, it is giving an error on the sequence.

Reply:
Put gitdiff after https://github.com/checkstyle/checkstyle/blob/7b7f92e3539465cb0828f5b1fd2680dd648e131f/azure-pipelines.yml#L167